apis/v1alpha2/backendtlspolicy_types.go

```diff
@@ -0,0 +1,156 @@
+
+	// TLS contains backend TLS policy configuration.
+	TLS *BackendTLSPolicyConfig `json:"tls"`
+}
+
+// BackendTLSPolicyConfig contains backend TLS policy configuration.
+// +kubebuilder:validation:XValidation:message="must not contain both CertRefs and StandardCerts",rule="(has(self.certRefs) && size(self.certRefs) > 0 && has(self.standardCerts) && self.standardCerts != ”)"
+// +kubebuilder:validation:XValidation:message="must specify either CertRefs or StandardCerts",rule="!(has(self.certRefs) && size(self.certRefs) > 0 || has(self.standardCerts) && self.standardCerts != ”)"
+type BackendTLSPolicyConfig struct {
+	// CertRefs contains one or more references to
+	// Kubernetes objects that contain PEM-encoded TLS certificates,
+	// which are used to establish a TLS handshake between the gateway
+	// and backend pod.
+	//
+	// If CertRefs is empty or unspecified, then StandardCerts must
+	// be specified.  Only one of CertRefs or StandardCerts may be
+	// specified, not both.
+	//
+	// If CertRefs is empty or unspecified, then system trusted
+	// certificates should be used. If there are none, or the
+	// implementation doesn't define system trusted certificates,
+	// then a TLS connection must fail.
+	//
+	// References to a resource in a different namespace are
+	// invalid.
+	//
+	// A single CertRef to a Kubernetes ConfigMap kind has "Core"
+	// support.  Implementations MAY choose to support attaching
+	// multiple certificates to a backend, but this behavior is
+	// implementation-specific.  Also implementation-specific is
+	// a CertRef of other object kinds, e.g. Secret.
+	//
+	// Support: Core - An optional single reference to a Kubernetes
+	// ConfigMap.
+	//
+	// Support: Implementation-specific (No reference, more than one
+	// reference, or resource types other than ConfigMaps.
+	// Service mesh may ignore.)
+	//
+	// +kubebuilder:validation:MaxItems=8
+	// +optional
+	CertRefs []v1beta1.ConfigMapObjectReference `json:"certRefs,omitempty"`
+
+	// StandardCerts specifies whether system CA certificates may
+	// be used in the TLS handshake between the gateway and
+	// backend pod.
+	//
+	// If StandardCerts is unspecified or set to "", then CertRefs must
+	// be specified with at least one entry for a valid configuration.
+	// If StandardCerts is unspecified or set to "", then CertRefs must
+	// be specified.  Only one of CertRefs or StandardCerts may be
+	// specified, not both.
+	//
+	// StandardCerts must be set to "System" when CertRefs is unspecified.
+	//
+	// If StandardCerts is set to "System", then the system trusted
+	// certificates should be used. If there are none, or the
+	// implementation doesn't define system trusted certificates,
+	// then a TLS connection must fail.
+	//
+	// Support: Core - An optional value to specify whether to use
+	// system certificates or not.
+	//
+	// Support: Implementation-specific (In the absence of support
+	// for usable system certs, may be ignored. Service mesh may ignore.)
```

Comment:
Not sure we need this? Are there implementations that would not be able to support some concept of system certs?

Reply:
I agree, let's leave this out and add it back if we need it.